lms/djangoapps/ccx/api/v0/views.py

```diff
@@ -152,6 +152,13 @@ def get_valid_input(request_data, ignore_missing=False):
     elif 'max_students_allowed' in request_data:
         field_errors['max_students_allowed'] = {'error_code': 'null_field_max_students_allowed'}
 
+    other_course_settings = request_data.get('other_course_settings')
+    if other_course_settings is not None:
```

Comment:
@Squirrel18 at the end we need to validate if exist in a different level, because when the request doesn't have the value, it returns the field error even though we don't need to add/change it

Reply:
In this case can't we use: if other_course_settings: and process it if it's present? @JuanDavidBuitrago